src/swap/SwapReviewScreen.tsx

```diff
@@ -69,33 +77,28 @@ export function SwapReviewScreen() {
   const { t } = useTranslation()
   const dispatch = useDispatch()
 
-  // Workaround for buying Celo
-  const CELO_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
-  const toCeloWorkAround = (token: string) => {
-    const symbol = coreTokens.find((token) => token.address === toToken)?.symbol
-    return symbol === 'CELO' ? CELO_ADDRESS : token
-  }
-
   // Token Symbols
   const toTokenSymbol = coreTokens.find((token) => token.address === toToken)?.symbol
   const fromTokenSymbol = coreTokens.find((token) => token.address === fromToken)?.symbol
 
+  // BuyAmount or SellAmount
+  const swapAmountParam = updatedField === Field.FROM ? 'sellAmount' : 'buyAmount'
+  const swapAmountInWei = multiplyByWei(swapAmount[updatedField]!)
+
   useEffect(() => {
     ValoraAnalytics.track(SwapEvents.swap_review_screen_open, {
       toToken,
       fromToken,
-      buyAmount: swapAmount[Field.TO] ?? '',
+      amount: swapAmount[updatedField] ?? '',
```

Comment:
why do we need this change? with this change, sometimes the `amount` will be a buy amount and sometimes a `sell` amount so swapping twice with the same value like 1 CELO : 5 cUSD, depending on which field you changed last the `amount` could be 1 or 5...?